docs/operator-guide/index.md

```diff
@@ -1,14 +1,21 @@
-# Welcome to SciCat Operator's Manual
+# Welcome to SciCat Operator's Guide
 
 ## Overview
 
-Getting SciCat up and running at your site should be rather straight forward for a test deployment. However turning into a production ready system may involve a bit more work, because different existing systems will need to be interfaced to SciCat.
+Getting SciCat up and running at your site should be rather straight forward for a test deployment. However turning into a production ready system may involve some more work, because different existing systems will need to be interfaced to SciCat.
 
-## Understanding the Components
+How to ingest metadata, set SciCat up to deploy it is best covered by understanding its core systems, backend and frontend, its features, configurations, and what else one could do to fully exploit all of SciCat's capabilities.
 
-For the subsequent sections it will be useful to have a "helicopter" overview of the various components that need to play together seamlessly. The following diagram shows these components and also shows potentially existing components at your site, that you would likely want to interface to SciCat. The specific diagram reflects essentially the situation at PSI as of Sept. 2020. Of course your situation may look different. The diagram should therefore be seen as an example, which you need to adapt to your situation.
+SciCat consists of a backend application, that is connected to the database - a MongoDB - and a frontend client exposing database content through a GUI to a user. At large scale facilities SciCat handles about 30 PB of data. 
 
-![ToDo: Updated schematic view of SciCat components](img/DacatDataflowV3.png)
+## Features 
+
+SciCat promises to cover these core aspects in a flexible way:
+
+1. Searchable metadata fields, most common and highly specific ones. Thereby, SciCat can answer the needs of several, not all, facilities running photon and neutron sources. Since the needs may differ many great features are configurable.
```

Comment:
Suggestion:

Replace from “Thereby…” to end with:

SciCat was developed by the PaNoSc community and has been successfully used more widely. This is because SciCat is highly configurable.